dispatch/apps/api/serializers.py

```diff
@@ -437,3 +437,17 @@ def update(self, instance, validated_data, action='update'):
             perform_action(self.context['request'].user.person, action, 'page', instance.pk)
 
         return instance
+
+class IntegrationSerializer(serializers.Serializer):
+
+    id = serializers.CharField(source='ID', read_only=True)
+    settings = serializers.JSONField(source='get_settings')
+
+    def save(self):
+
+        settings = self.validated_data.get('get_settings', None)
```

Comment:
None is the default for `dict.get(nonexistent_key)`. Are you being explicit on purpose?